tests/test_vision_data.py

```diff
@@ -7,17 +7,36 @@ def path():
     path = untar_data(URLs.MNIST_TINY)
     return path
 
+def prep_mnist_tiny_csv_labels(path):
```

Comment:
@sgugger : should we update `mnist_tiny/labels.csv` to have 3/7 labels (instead of 0/1 it currently has) 

that ll get rid of this extra function I had to put while trying to make all tests call the same sanity_check (instead of repeating the same check in different fashions)